ports/stm32/powerctrlboot.c

```diff
@@ -104,6 +104,77 @@ void SystemClock_Config(void) {
     powerctrl_config_systick();
 }
 
+#elif defined(STM32G0)
+
+void SystemClock_Config(void) {
+    // Enable power control peripheral
+    __HAL_RCC_PWR_CLK_ENABLE();
+
+    // Set flash latency to 2 because SYSCLK > 48MHz
+    FLASH->ACR = (FLASH->ACR & ~0x7) | 0x2;
+
+    #if MICROPY_HW_CLK_USE_HSI
```

Comment:
Is there a reason you didn't implement HSE support?  Just for simplicity?

Reply:
Yes, I just wanted to go for a minimal implementation.
I figured I could add HSE later.

Reply:
OK